geant4.sh

```diff
@@ -40,11 +40,30 @@ cmake $SOURCEDIR                                    \
   -DCMAKE_STATIC_LIBRARY_C_FLAGS="-fPIC"            \
   -DGEANT4_USE_G3TOG4=ON                            \
   -DGEANT4_INSTALL_DATA=ON                          \
-  -DGEANT4_USE_SYSTEM_EXPAT=OFF
+  -DGEANT4_USE_SYSTEM_EXPAT=OFF                     \
+  ${XERCESC_ROOT:+-DGEANT4_USE_OPENGL_X11=ON -DGEANT4_USE_GDML=ON -DXERCESC_ROOT_DIR=$XERCESC_ROOT}
```

Comment:
Why do you need the OpenGL bindings? what happens if they are not found? What happens if this is built on mac? Why do you need GDML?